torch/jit/__init__.py

```diff
@@ -463,149 +463,14 @@ def wrapper(func):
         for name in executor_options:
             executor_options[name] = kwargs.pop(name, executor_options[name])
```

Comment:
Can you add a check that `kwargs` are empty after this loop? I just noticed we forgot about it, and will silently ignore anything we don't support (e.g. names with typos)